src/pages/Selector/transform-employees.js

```diff
@@ -0,0 +1,23 @@
+export const alphabeticalSort = data => {
```

Comment:
ahh I was thinking more like move this function to a separate file with something like:

https://www.cpubenchmark.net/cpu.php?cpu=Intel+Core+i7-8750H+%40+2.20GHz&id=3237

```
export const alphabeticalSort = (data, accessor) => {
  return data.sort((x, y) => {
   let xValue = accessor(x);
   let yValue = accessor(y); 

   if (xValue < yValue) {
      return -1;
    }

    if (xValue > yValue) {
      return 1;
    }

    return 0;
  });
};
```

and then import it here.

Just for reusability.